src/index.ts

```diff
@@ -1,11 +1,14 @@
-const cacheManager = require('cache-manager')
+import * as cacheManager from 'cache-manager'
+import {createApp} from './github-app'
+import {createRobot, Robot} from './robot'
+import {createServer} from './server'
+import {resolve} from './resolver'
+import {logger} from './logger'
+import {createWebhookProxy} from './webhook-proxy'
+import * as express from 'express'
+import * as Logger from 'bunyan'
```

Comment:
remove this import, you should use this here

Reply:
@ocombe I ended up keeping this and just doing `export { Logger }` to get the compiler to stop warning me that I wasn't naming the export:

https://github.com/probot/probot/blob/920d5b67f64d54d471302836f3d59a40814f604c/src/index.ts#L116

Does that make sense here? I couldn't get any of the other workaround we discussed to work.